README.md

```diff
@@ -33,6 +33,7 @@ Server installation. Simplified. :cloud:
 * Report [your issue](https://github.com/nextcloud/vm/issues/new)
 * Help us with [existing issues](https://github.com/nextcloud/vm/issues)
 * Write scripts so that the release process becomes automated with [Vagrant](https://www.vagrantup.com/docs/getting-started/), [Terraform](https://www.terraform.io/) or similar
+* Help us testing scripts, that aren't ready for the release version, yet. Please look at [this subfolder](https://github.com/nextcloud/vm/master/not-supported) for more infos.
```

Comment:
Released
Info
Remove the comma after scripts and yet

Reply:
tried to address this